doc/api/repl.md

```diff
@@ -411,6 +417,8 @@ added: v0.1.91
     `SIGINT` is received, i.e. `Ctrl+C` is pressed. This cannot be used together
     with a custom `eval` function. Defaults to `false`.
```

Comment:
`eval` is using in multiple places as an abstraction. It is just in the code that it shadows global.